src/Microsoft.ApiDesignGuidelines.Analyzers/Core/ImplementStandardExceptionConstructors.cs

```diff
@@ -6,18 +6,29 @@
 using Analyzer.Utilities;
 
 namespace Microsoft.ApiDesignGuidelines.Analyzers
-{                   
+{
     /// <summary>
-    /// CA1032: Implement standard exception constructors
+    /// CA1032 - redefined: Implement standard exception constructors
+    /// Cause: A type extends System.Exception and does not declare all the required constructors. 
+    /// Description: Exception types must implement the following constructors. Failure to provide the full set of constructors can make it difficult to correctly handle exceptions
+    /// For CSharp, example when type name is GoodException 
+    ///     public GoodException()
+    ///     public GoodException(string)
+    ///     public GoodException(string, Exception)
+    /// For Basic, example
+    ///     Sub New()
+    ///     Sub New(message As String)
+    ///     Sub New(message As String, innerException As Exception)
+    /// Redefined - because, in the original FxCop rule, it was also checking for a 4th constructor, as listed below, which after discussion with Sri was decided as covered by other analyzers
```

Comment:
The reason we shouldn't check for the fourth constructor is that ISerializable types are not available in .NET Core and hence an exception cannot have that constructor.